benchmark/harness_results.go

```diff
@@ -43,33 +48,33 @@ func (r *BenchResult) EvergreenPerfFormat() ([]interface{}, error) {
 
 	out := []interface{}{
 		map[string]interface{}{
-			"name": r.Name + "-throughput",
-			"results": map[string]interface{}{
-				"1": map[string]interface{}{
-					"seconds":        r.Duration.Round(time.Millisecond).Seconds(),
-					"ops_per_second": r.getThroughput(median),
-					"ops_per_second_values": []float64{
-						r.getThroughput(min),
-						r.getThroughput(max),
-					},
+			"info": map[string]interface{}{
+				"test_name": r.Name + "-throughput",
+				"args": map[string]interface{}{
+					"threads": 1,
 				},
 			},
+			"metrics": []Metric{
+				{Name: "seconds", Value: r.Duration.Round(time.Millisecond).Seconds()},
+				{Name: "ops_per_second", Value: r.getThroughput(median)},
+				{Name: "ops_per_second_values", Value: []float64{r.getThroughput(min), r.getThroughput(max)}},
```

Comment:
Nice, I see data populating in the [Trend Charts](https://spruce.mongodb.com/task/mongo_go_driver_perf_perf_patch_8a397a401000e6ba266e8a18beabc8af4da4a773_616f31e63e8e8608fdd848e8_21_10_19_21_00_24/trend-charts?execution=0) tab on the most recent patch run.

The [Raw performance results](https://cedar.mongodb.com/rest/v1/perf/task_id/mongo_go_driver_perf_perf_patch_8a397a401000e6ba266e8a18beabc8af4da4a773_616f31e63e8e8608fdd848e8_21_10_19_21_00_24) indicate the `ops_per_second_values` are nil.

I tried running one test case locally by applying this change:

```
diff --git a/benchmark/harness_main.go b/benchmark/harness_main.go
index 5d1db3d6..36094539 100644
--- a/benchmark/harness_main.go
+++ b/benchmark/harness_main.go
@@ -58,7 +58,7 @@ func DriverBenchmarkMain() int {
 }
 
 func runDriverCases(ctx context.Context) []*BenchResult {
-	cases := getAllCases()
+	cases := getAllCases()[0:1]
 
 	results := []*BenchResult{}
 	for _, bc := range cases {

```

Running `go run cmd/godriver-benchmark/main.go` produces a `perf.json` showing expected output:

```
[
   {
      "info": {
         "args": {
            "threads": 1
         },
         "test_name": "CanaryIncCase-throughput"
      },
      "metrics": [
         {
            "name": "seconds",
            "value": 5.003
         },
         {
            "name": "ops_per_second",
            "value": 3543341892.2756743
         },
         {
            "name": "ops_per_second_values",
            "value": [
               3592170791.915604,
               140450400.83982596
            ]
         }
      ]
   }
]
```

Perhaps only a single value is supported in a metric? I suggest splitting this into two metrics:
`ops_per_second_min` and `ops_per_second_max` and rerunning the perf task to see if that applies.